src/laserdisc/LaserdiscPlayer.cc

```diff
@@ -679,6 +680,16 @@ void LaserdiscPlayer::autoRun()
 		return;
 	}
 
+	auto *player = motherBoard.getCassettePort().getCassettePlayer();
```

Comment:
In the general case `getCassettePlayer()` can return `nullptr`.
But I don't know if in this specific context that's possible. May I suggest to either:
* Change the line below to `if (player && player->...)`.
* Or add a `assert(player); // short reasoning for why this must be the case`.